README.md

```diff
@@ -64,7 +64,16 @@ These packages will install when you install Terminus.
 
 `*** TBD ***`
 
+## Testing
 
+To run the functional tests:
+
+1. If you need the basics (php, git, direnv, etc): `brew bundle install`
+2. For code coverage: `pecl install pcov`
```

Comment:
for what it's worth, this was required for them to run, at least when I first got them to run and I can't see why it would have changed.